src/solidClientHelpers/resource.js

```diff
@@ -107,12 +107,12 @@ export async function saveResource({ dataset, iri }, fetch) {
 }
 
 export async function deleteResource(resource, policiesContainer, fetch) {
-  const { dataset, iri } = resource;
+  const { info, iri } = resource;
```

Comment:
Instead of packaging a custom object with `info` and `iri`, I think you should send the resourceInfo object, and use `getSourceUrl` to get the `iri`.